UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -148,6 +149,8 @@ public partial class GameManager : MonoBehaviour, IInitialise
 	[NonSerialized] public int errorCounter;
 	[NonSerialized] public int uniqueErrorCounter;
 
+	[HideInInspector] public Techweb TechwebServer;
```

Comment:
Looks like `Techweb` is a singleton. Why do you need a reference in both places?